src/client/controller/RoomsController.js

```diff
@@ -45,6 +46,12 @@ RoomsController.prototype.attachEvents = function()
     this.repository.on('room:players', this.applyScope);
     this.repository.on('room:game', this.applyScope);
 
+    var self = this;
+    this.repository.on('rooms:nbPlayers', function(e) {
```

Comment:
Make it a real method `onTotalPlayers` (or something like that) instead of an anonymous function.
It's more clean and allows you to properly remove the event.